unit_tests/test_exceptions.py

```diff
@@ -62,6 +62,17 @@ def test_hit_w_content_as_str(self):
         self.assertEqual(exception.message, 'Not Found')
         self.assertEqual(list(exception.errors), [])
 
+    def test_hit_w_content_as_unicode(self):
+        from google.cloud.exceptions import NotFound
+        response = _Response(404)
+        content = u'{"error": {"message": "That\u2019s all we know."}}'
+        exception = self._callFUT(response, content)
+        self.assertTrue(isinstance(exception, NotFound))
+
+        self.assertEqual(exception.message, u'That\u2019s all we know.')
```

Comment:
You could re-use this value (`u'That\u2019s all we know.'`) and make the test more modular (DRY)